tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/PriceRepositoryTests.cs

```diff
@@ -63,11 +63,59 @@ public async Task GetPricesBySolutionIdQueryAsync_ValidSolutionId_ReturnsPriceLi
             request.Count().Should().Be(2);
         }
 
-        private static async Task InsertPriceAsync(string solutionId)
+        [Test]
+        public async Task GetPricingByPriceIdQueryAsync_InvalidPriceId_ReturnsEmptyList()
```

Comment:
Can you change it the name of the test to start with the name of the method please
GetPriceByPriceIdQueryAsync